napari/components/_layer_slicer.py

```diff
@@ -21,7 +21,12 @@ class _LayerSlicer:
     Events
     ------
     ready
-        emitted after slicing is done with a dict value that maps from layer to slice response
+        emitted after slicing is done with a dict value that maps from layer
+        to slice response. Note that this may or may not be emitted on a
+        non-main thread. If usage of this event relies on something happening
```

Comment:
This double negation make it unclear for me.

Reply:
Sorry about that! I can update it.

Reply:
Is that better?